src/App.jsx

```diff
@@ -1,18 +1,33 @@
 import "./App.css";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
 import Header from "./components/Header";
-import ArticleStateWrapper from "./components/ArticleStateWrapper";
 import HomePage from "./pages/HomePage";
 import TopicsPage from "./pages/TopicsPage";
+import SingleArticlePage from "./pages/SingleArticlePage";
+import Navbar from "./components/Navbar";
+import { useState } from "react";
+import CommentsPage from "./pages/CommentsPage";
 
 function App() {
+  const [articles, setArticles] = useState([]);
```

Comment:
Cant see anything else thats using this state - so probably doesnt need to be in app